Superposition/Tasks.qs

```diff
@@ -129,21 +129,31 @@ namespace Quantum.Kata.Superposition {
     }
 
 
-    // Task 10. |00⟩ + |01⟩ + |10⟩ state
+    // Task 10. Superposition of all even numbers
+    // Input: N qubits in |0...0⟩ state.
+	// Goal: create a superposition of all even numbers on N qubits.
```

Comment:
This line and lines 135 and 137 still have tabs in them (you can see GitHub indents them the same as line 136 even though 136 should be the next level of indentation)